converter/config/yaml/yaml_test.go

```diff
@@ -43,13 +43,41 @@ Output:
     # This database holds information about RITA managed databases.
     MetaDB: MetaDatabase
 
-IPFIX:
-  # CIDR ranges of networks to mark local
-  LocalNetworks:
-    - 192.168.0.0/16
-    - 172.16.0.0/12
-    - 10.0.0.0/8
-    - 195.154.15.32`
+Filtering:
+    # These are filters that affect which flows are processed and which
+    # are dropped.
+    # A good reference for networks you may wish to consider is RFC 5735.
+    # https://tools.ietf.org/html/rfc5735#section-4
+
+    # Example: AlwaysInclude: ["192.168.1.2/32"]
+    # This functionality overrides the NeverInclude and InternalSubnets
+    # section, making sure that any connection records containing addresses from
+    # this range are kept and not filtered.
+    AlwaysInclude: []
```

Comment:
Can we change this to look more like the other two fields? Something like 
AlwaysInclude:
    # - 127.0.0.1
I think will help aid the understanding that we can keep the same format as below and don't need to comma separate

Reply:
Sure!

Reply:
Done